app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -214,20 +214,28 @@ var update_registry = function() {
   $('#total_cost').html(tc + ' ' + denomination);
   $('#total_net').html(net + ' ' + denomination);
 
+  let transactions = [];
+
+  first_transaction = {
```

Comment:
do we want to add something to the UI to let the funder toggle whether/not they want  to cancel the bounty?  what should that look like? @willsputra @PixelantDesign

Reply:
thanks for the detailed answer.. 

@PixelantDesign we had a convo yesterday about whether or not to re-introduce the checkbox..

Reply:
> Default to paying from escrow since this is what users expect.

I just dont know that theres any way to pay from escrow without re-adding that checkbox.  Since Tips/bounties are two seperate systems

Reply:
Also, just for my clarification, is `pay from escrow` in reference to: 
1. The old way where users are refunded and then a new transaction made.
2. Or, where the new function is made to have payments directly subtracted from the escrow amount?

Sorry, sorry - just clarifying for my part :)

Reply:
@owocki I was not aware that we could not by default pay from escrow. If that is the case, we'll need a check box that says 

- [ ] Pay using bounty funds

Pay from escrow to me means automatically pulling from the bountied funds.
Defer to @owocki @mbeacom on the implementation question from @Anish-Agnihotri. 

cc: @ceresstation

Reply:
OK yes I think that we need to add the checkbox back @Anish-Agnihotri

Reply:
Alright, I'll be adding that checkbox in and pushing a PR with <24hr ETA. 

I'm actually at a conference today and all of my pending PR's that have been in limbo for the past month are held up.

Reply:
hows it going?

Reply:
Thanks @Anish-Agnihotri 

My only comment is **please show the orange alert anytime checkbox is unchecked.**
It should say: Please note additional funds will be withdrawn from your Metamask wallet.


Updating the thread here for transparency: 

Anish Agnihotri [10:54 PM]
Hey, I’m working on advanced payout atm and getting it knocked out. Basically, the integration I’m going for is now:

1. If the checkbox is not checked, and total amount is under 100% payout as tip from your own wallet.

2. If the checkbox is not checked, and total amount is over 100%, payout as tip from your wallet, but also give an additional checkbox option (in the orange alert) to close bounty.

3. If the checkbox is checked, pay amount of tips from escrow amount first, and then if escrow is remaining keep task open, and if not - close the task.